tools/run_tests/xds_k8s_test_driver/kubernetes-manifests/client-secure.deployment.yaml

```diff
@@ -41,6 +41,10 @@ spec:
             value: "true"
           - name: GRPC_XDS_EXPERIMENTAL_V3_SUPPORT
             value: "true"
+          - name: GRPC_VERBOSITY
+            value: "DEBUG"
+          - name: GRPC_TRACE
+            value: "xds_client,xds_resolver,xds_cluster_manager_lb,cds_lb,xds_cluster_resolver_lb,priority_lb,xds_cluster_impl_lb,weighted_target_lb"
```

Comment:
I think it's better to [burn these](https://docs.docker.com/engine/reference/builder/#env) in concrete language-specific docker images. I don't think these are a part of standard contract, and will apply to java/go/nodejs.

Here's how I added a language-specific env var in java:
https://github.com/grpc/grpc-java/blob/869b395ec02b771b4711aaae04337a63b474be71/buildscripts/xds-k8s/test-client.Dockerfile#L12

Reply:
Done. For both Python and C++.